README.md

```diff
@@ -78,18 +79,39 @@ For example to open a terminal to an SSH connection that you have created:
 
 ```go
 	// session is an *ssh.Session from golang.org/x/crypto/ssh
-	// win is a fyne.Window created to hold the content
 	in, _ := session.StdinPipe()
 	out, _ := session.StdoutPipe()
-
 	go session.Run("$SHELL || bash")
-
+	
+	// run new terminal and close app on terminal exit.
 	t := terminal.New()
-	w.SetContent(t)
-
 	go func() {
 		_ = t.RunWithConnection(in, out)
 		a.Quit()
 	}()
+
+	// OPTIONAL: dynamically resize the terminal session
+	cell := canvas.NewText("M", color.White)
+	cell.TextStyle.Monospace = true
+	cellHeight := int(math.Round(float64(cell.MinSize().Height)))
```

Comment:
I'm not sure why this is being rounded here. A Fyne size is floating point because it can refer to many pixels. If you round or truncate here you are removing the ability for us to handle HiDPI properly...

Reply:
This is copying the guessCellSize() function in term.go.  Which is what func (t *Terminal) Resize(s fyne.Size) uses.
```go
func (t *Terminal) guessCellSize() fyne.Size {
	cell := canvas.NewText("M", color.White)
	cell.TextStyle.Monospace = true

	min := cell.MinSize()
	return fyne.NewSize(float32(math.Round(float64(min.Width))), float32(math.Round(float64(min.Height))))
}
```
From my experimentation not rounding results in extra padding for the whole terminal.  I think it's because canvas text itself is padded.
The highest resolution monitor I own is 1080p so things could be padding poorly on 4k right now since it's already being used by func (t *Terminal) Resize(s fyne.Size).

If it is not currently rendering correctly to the window size on HiDPI displays right now then it would be a good idea to modify guessCellSize().

Reply:
The code you copied from does n to apply int/rounding as far as I can tell...